versioned_docs/version-0.4.0/register-to-try.mdx

```diff
@@ -32,5 +32,19 @@ playground contains three kind clusters.
 You will receive an expiration-reminder email an hour (60 minutes) before your access expires. 
 :::
 
+Now with the access to your sandbox virtual machine (VM), a few tasks that you can do are:
+
+* Run `kubeslice-cli install --profile=full-demo` to get a full environment setup for you. 
+  You can use that to explore and learn about KubeSlice without installing anything.
+* If you want to practice installing Kubeslice, you can do the 
+  [minimal-demo](/versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx).    
+* If you are a contributor, you can install a topology, clone your repo to the sandbox VM, and 
+  try out your modifications.
+
+  :::warning
+  Remember that the sandbox environment expires in **four hours**, which can erase all the local content if you don't save it. 
```

Comment:
...will erase... (not "can"... no doubt about it... it will be erased).

Reply:
done